src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
At this point, just edit your files to replace the 2016 license, then commit. It looks like your master branch (at least at https://github.com/kannon92/psi4/commits/cholesky_improvements) is not up to date with respect to the upstream master as the most recent non-this-PR commit is from 17 May. So you'll want to merge (or even better, rebase) from upstream. Assuming your `git remote -v` has both `origin` and `upstream`, on your local cholesky_improvements with everything checked in, you can `git pull --rebase upstream master`. This will make your branch look like upstream master plus your commits at the top. When you push to github, you'll have to `git push --force origin cholesky_improvements`. Note that force pushing is only appropriate because it only to `your` branch. Travis will recommence.